### ISSUE TITLE
widget

### ISSUE THREAD

**Issue by Backkevin** [iobroker-community-adapters/ioBroker.coronavirus-statistics#76]
Platform
    Windows
Betriebssystem
    win32
Architektur
    x64
CPUs
    16
Geschwindigkeit
    2800 MHz
Modell
    Intel(R) Xeon(R) CPU X5560 @ 2.80GHz
RAM
    23.99 GB
System Betriebszeit
    00:12:37
Node.js
    v12.18.3
NPM
    6.14.6

my location are not working in widget 
Luxembourg 
widget is empty but data exist in iobroker
![Untitled](https://user-images.githubusercontent.com/6032786/92175896-bc6e9180-ee3d-11ea-9196-1695c2f01512.png)

coronavirus-statistics.0	2020-09-03 23:37:00.156	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.155	warn	(6280) Object coronavirus-statistics.0.Luxembourg.criticalPerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, fi
coronavirus-statistics.0	2020-09-03 23:37:00.148	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.148	warn	(6280) Object coronavirus-statistics.0.Luxembourg.recoveredPerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, f
coronavirus-statistics.0	2020-09-03 23:37:00.140	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.140	warn	(6280) Object coronavirus-statistics.0.Luxembourg.activePerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file
coronavirus-statistics.0	2020-09-03 23:37:00.134	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.133	warn	(6280) Object coronavirus-statistics.0.Luxembourg.oneTestPerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, j
coronavirus-statistics.0	2020-09-03 23:37:00.126	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.126	warn	(6280) Object coronavirus-statistics.0.Luxembourg.oneDeathPerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file,
coronavirus-statistics.0	2020-09-03 23:37:00.120	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.119	warn	(6280) Object coronavirus-statistics.0.Luxembourg.oneCasePerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, j
coronavirus-statistics.0	2020-09-03 23:37:00.107	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.106	warn	(6280) Object coronavirus-statistics.0.Luxembourg.population is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
coronavirus-statistics.0	2020-09-03 23:37:00.061	warn	(6280) This object will not be created in future versions. Please report this to the developer.
coronavirus-statistics.0	2020-09-03 23:37:00.061	warn	(6280) Object coronavirus-statistics.0.Luxembourg.todayRecovered is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, jso
coronavirus-statistics.0	2020-09-03 23:36:59.930	warn	(3200) Terminated (ADAPTER_ALREADY_RUNNING): Without reason

**Comment by DutchmanNL** [comment]
which version are u using ? These log error have been fixed already in previous version

**Comment by Backkevin** [comment]
Installierte Version: 0.6.0

and on some PCs the Widget Show noting

________________________________
Von: Dutchman <notifications@github.com>
Gesendet: Dienstag, 8. September 2020 12:37
An: iobroker-community-adapters/ioBroker.coronavirus-statistics <ioBroker.coronavirus-statistics@noreply.github.com>
Cc: Backkevin <back_kevin@hotmail.com>; Author <author@noreply.github.com>
Betreff: Re: [iobroker-community-adapters/ioBroker.coronavirus-statistics] widget (#76)


which version are u using ? These log error have been fixed already in previous version

**Comment by FRYYYYYY** [comment]
I have the same error messages in the log, I just installed the adapter today. Apparently wrong object types are set:

```
2020-11-03 09:46:24.272 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.todayRecovered is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
--
2020-11-03 09:46:24.273 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.609 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.population is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.609 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.687 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.oneCasePerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.687 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.705 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.oneDeathPerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.706 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.724 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.oneTestPerPeople is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.725 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.785 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.activePerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.785 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.801 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.recoveredPerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.801 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
2020-11-03 09:46:24.866 - warn: coronavirus-statistics.0 (11428) Object coronavirus-statistics.0.Germany.criticalPerOneMillion is invalid: obj.common.type has an invalid value (value) but has to be one of number, string, boolean, array, object, mixed, file, json
2020-11-03 09:46:24.866 - warn: coronavirus-statistics.0 (11428) This object will not be created in future versions. Please report this to the developer.
```

**Comment by DutchmanNL** [comment]
fixed in >0.6.2
